toolkit/tools/imagecustomizer/container/build-container.sh

```diff
@@ -17,13 +17,15 @@ function showUsage() {
     echo "build-container.sh \\"
     echo "    -t <container-tag>"
     echo "    -a <architecture> (default: amd64)"
+    echo "    -c <azure-monitor-connection-string> (optional)"
     echo
 }
 
-while getopts "t:a:b" OPTIONS; do
+while getopts "t:a:c:b" OPTIONS; do
   case "${OPTIONS}" in
     t ) containerTag=$OPTARG ;;
     a ) ARCH=$OPTARG ;;
+    c ) azmonitor_connection_string=$OPTARG ;;
```

Comment:
The variable naming scheme is a little inconsistent in this file. But currently both camelCase and SCREAM_CASE is being used. This variable is adding snake_case into the mix.

Reply:
updated to use camelCase for the variable